peekaboo/config.py

```diff
@@ -457,6 +481,7 @@ def __init__(self, config_file=None):
         self.cuckoo_maximum_job_age = 15*60
 
         self.cortex_url = 'http://127.0.0.1:9001'
+        self.cortex_tlp = tlp.AMBER
```

Comment:
I like the new default of amber as a compromise. It leaves the admin the option to either configure Peekaboo to match the Cortex analyzers they emply or configure Cortex analyzers relative to our amber setting according to their preference.